### PR TITLE
Hotfix/fix deploy v2

### LINKED ISSUE #132: Restructure the git repository

### LINKED ISSUE #140: Automate deploy process for the new git structure

### PR DESCRIPTION
**Pull request by arjadas**
Added a util module so that other directories can access utils globally. that was required to run github actions for the three applications as a consequence, we also had a better overall structure. Hypothetically we could also use utils in the admin-console if we decide to develop further.

### CODE REVIEW

**Review by adendiamond — APPROVED** [review]
👍👍

**Comment by TharathepC** [comment]
Nice work!

File: .github/workflows/deploy-to-cloudlfare-pages.yaml [hunk 1]
```diff
@@ -0,0 +1,175 @@
+name: Deploy to Cloudflare Pages
```
Comment (TharathepC) [inline .github/workflows/deploy-to-cloudlfare-pages.yaml:1]: Looks great

**Review by TharathepC — APPROVED** [review]
Nice

**Pull request merged**